presto-tests/src/test/java/com/facebook/presto/tests/TestLocalBinarySpilledQueries.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.SystemSessionProperties;
+import com.facebook.presto.metadata.SessionPropertyManager;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.tpch.TpchConnectorFactory;
+import com.facebook.presto.tpch.testing.SampledTpchConnectorFactory;
+import com.google.common.collect.ImmutableMap;
+
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static com.facebook.presto.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+
+public class TestLocalBinarySpilledQueries
+        extends AbstractTestApproximateQueries
+{
+    private static final String TPCH_SAMPLED_SCHEMA = "tpch_sampled";
+
+    public TestLocalBinarySpilledQueries()
+    {
+        super(createLocalQueryRunner(), createDefaultSampledSession());
+    }
+
+    private static LocalQueryRunner createLocalQueryRunner()
+    {
+        Session defaultSession = testSessionBuilder()
+                .setCatalog("local")
+                .setSchema(TINY_SCHEMA_NAME)
+                .setSystemProperties(ImmutableMap.of(
+                        SystemSessionProperties.OPERATOR_MEMORY_LIMIT_BEFORE_SPILL, "100MB",
+                        SystemSessionProperties.MAX_ENTRIES_BEFORE_SPILL, "8"))
+                .build();
```

Comment:
you can add a comment explaining motivation behind this configuration. That you wanted to make query spill very often.